content/en/service_management/app_builder/queries.md

```diff
@@ -17,7 +17,7 @@ Queries populate your app with data from Datadog APIs or supported integrations.
 
 The [Action Catalog][10] within the Datadog App provides actions that can be performed as queries against your infrastructure and integrations using App Builder. You can orchestrate and automate your end-to-end processes by linking together actions that perform tasks in your cloud providers, SaaS tools, and Datadog accounts.
 
-To add a query, click the plus (**+**) icon in the **Queries** section and search for an action to add to your app. After you've added the query action, it appears in the query list above the query editor. Click and drag queries to reorder them. Select a query to configure it.
+To add a query, click the Data (<i class="icon-api"></i>) icon to open the Data tab, click the plus (**+**), select **Query**, and search for an action to add to your app. After you've added the query action, it appears in the **Actions** List. Select a query to configure it.
```

Comment:
I noticed this is rendering weirdly on the hash link -
<img width="207" alt="Screenshot 2025-01-17 at 2 50 01 PM" src="https://github.com/user-attachments/assets/c5020b40-aac6-47ac-b5a7-f755fb832904" />